src/stratis_cli/_error_reporting.py

```diff
@@ -92,6 +94,13 @@ def interpret_errors(errors):
         if isinstance(error, DbusClientMissingPropertyError):
             return _DBUS_INTERFACE_MSG
 
+        if isinstance(error, StratisCliStratisdVersionError):
+            fmt_str = (
+                "The version of stratisd you are using %s is not compatible "
+                "with this version of stratis. stratis can execute only the "
+                "subset of its commands that do not require stratisd.")
```

Comment:
how about something like "stratis can only run help and version-related commands" for the second sentence there?

Reply:
also, maybe include min/max versions stratis-cli can handle in the output?

Reply:
> how about something like "stratis can only run help and version-related commands" for the second sentence there?

We don't want to be too specific here, or the text will rapidly get out of date. For example, I think it's possible to also query redundancy, and in future maybe more. But this message can certainly be more informative.